src/app/modules/courses/components/courses/courses.component.ts

```diff
@@ -10,9 +10,11 @@ import { mergeMap, map } from 'rxjs/operators';
   templateUrl: './courses.component.html',
   styleUrls: ['./courses.component.scss']
 })
-export class CoursesComponent implements OnInit {
+export class CoursesComponent implements OnInit, OnDestroy {
   categories: any[];
```

Comment:
categories and courses should have types

Reply:
we should avoid using any when possible since the whole point of typescript is to type things